packages/components/src/BaseStyleSheet.scss

```diff
@@ -37,19 +37,6 @@ body {
   }
 }
 
-#preload-fonts {
```

Comment:
enterprise uses this class, unless you have a fix to remove it from enterprise, but really the same preload fix should happen in it's app loader too, 

note that preloading is required not just before monaco init, but also any iris-grid init.

Reply:
When updating Enterprise to the latest published package I'll add the fix then, as it's a breaking change.